Objective-C/Internal/Replicator/CBLWebSocket.mm

```diff
@@ -376,12 +376,12 @@ - (void) _sendWebSocketRequest {
     
     slice sessionCookie = _options[kC4ReplicatorOptionCookies].asString();
     if (sessionCookie.buf)
-        [_logic addValue: sessionCookie.asNSString()  forHTTPHeaderField: @"Cookie"];
+        [_logic setValue: sessionCookie.asNSString()  forHTTPHeaderField: @"Cookie"];
     
     
     NSString* cookie = [_db getCookies: _remoteURL];
     if (cookie.length > 0)
-        [_logic addValue: cookie  forHTTPHeaderField: @"Cookie"];
+        [_logic setValue: cookie  forHTTPHeaderField: @"Cookie"];
```

Comment:
How does this change work with multiple Cookies (one from kC4ReplicatorOptionCookies and the other from the cookie store)?

Reply:
it should ideally comes like cookieOne=value; cookieTwo=value2

Reply:
From the doc that you have linked:
```
The new value for the header field. Any existing value for the field is replaced by the new value.
```
Do you mean Cookie key is a special key that setting multiple values will be combined with semi-colons?

Reply:
if same cookie-key is used, it will override the previous set cookie.

Reply:
I get what you mean

Reply:
OK